boards/common/microbit/Kconfig

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2020 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+config MODULE_MICROBIT
+    bool "Microbit Support Module"
```

Comment:
Does this one have to be visible? Should it `default y`?

Reply:
Yes its what enables the `microbit` led matrix-related code, Ill change the message